go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/ceph/ceph-csi
 go 1.13
 
 require (
+	github.com/aws/aws-sdk-go v1.35.24
```

Comment:
the latest available release is 1.38.8 can we use that?

Reply:
I don't know, this was picked automatically by `go mod`?

Reply:
will update to take the most current version